api/v1/apicurioregistry_types.go

```diff
@@ -81,10 +82,8 @@ type ApicurioRegistrySpecConfigurationSecurity struct {
 }
 
 type ApicurioRegistrySpecConfigurationSecurityHttps struct {
-	Enabled     bool   `json:"enabled,omitempty"`
+	DisableHttp bool   `json:"disableHttp,omitempty"`
```

Comment:
this is going to result in an API breaking change, correct @jsenko ?

Reply:
No because this feature has not been present in a previous release.

Reply:
Oh, wow! thanks! Where can I find the last released version of this file?